.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
```

Comment:
Should we add `pull_request` and `workflow_dispatch` triggers?

Reply:
I have added a minimal workflow to revive a testing in check's project. If you have an energy and spare time, you can polish suggested workflow for better state.

Reply:
Ok, no problem, we can add it later.